code/datums/character_preview.dm

```diff
@@ -106,6 +106,9 @@ datum/character_preview
 		src.preview_mob.real_name = "clone of " + name
 		src.preview_mob.name = "clone of " + name
 
+	proc/mirror_appearance(mob/living/carbon/human/to_copy) // This is like using a sledgehammer in a surgery where a scapel is better. You could directly change preview_mob, change it's clothes and appearance safely and sleep like someone who didn't commit a felony, or you can use this.
```

Comment:
This comment is not useful to the next coder, imo. Is there actually a problem with using the proc? If so, don't make it and do it the way you would recommend. If there isn't a problem, is the comment needed?